### ISSUE TITLE
[FEATURE] User Plugin Setting for "Enable on mobile"

### ISSUE THREAD

**Issue by hugomarins** [bjsi/incremental-everything#27]
What I am asking I think should be the default procedure for all plugins (I posted a request in [RemNote feedback site](https://feedback.remnote.com/p/plugin-setting-for-enable-on-mobile)).

In the plugin manifest, the author of the plugin chooses if it should be available on mobile ("enableOnMobile": true/false,).

What I would like is the user itself being able to choose if a plugin (that the author made enabled for mobile) will or will not be enabled on mobile for him.

Some plugins (like this one [Incremental Everything] and Queue Time Tracker) may not be so good to use on mobile, and the user having the possibility of choosing in the plugin settings if that plugin should run on mobile for him would be very nice.

**Comment by WandersonRossi** [comment]
Yes! I want this too! It may be a user preference if they want to use a certain plugin on mobile or not. This is one good example of plugin that I wish could be disabled just on mobile, but kept enabled on pc!
